packages/mdc-textfield/index.js

```diff
@@ -79,7 +82,41 @@ export class MDCTextfield extends MDCComponent {
           label.classList.remove(className);
         }
       },
-    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_()));
+      eventTargetHasClass: (target, className) => target.classList.contains(className),
+      registerTextFieldClickHandler: (handler) => this.root_.addEventListener('click', handler),
+      deregisterTextFieldClickHandler: (handler) => this.root_.addEventListener('click', handler),
```

Comment:
Copypasta! `addEventListener` -> `removeEventListener`.

This seems like something tests should catch...